src/add_remove.js

```diff
@@ -0,0 +1,83 @@
+import { fromStorage, toStorage, reloadStore } from './store';
+// eslint-disable-next-line import/no-cycle
+import { dragHover } from './dragdrop';
+import { refreshStore } from './status';
+
+const editHandlers = () => {
+  const todoList = document.getElementsByClassName('todo-item');
+  for (let i = 0; i < todoList.length; i += 1) {
+    const labelElem = todoList[i].children[0].children[1];
+    labelElem.addEventListener('input', () => {
```

Comment:
Hi, This is an additional review as a result of the second opinion request.

The CR meant when the user is trying to edit a task, the keypress `Enter` is saved in the localStorage, and also created a new line on the UI.

To reproduce this error: try to edit one of the tasks, and press `Enter` key.

Kindly check the screenshot from your app below;
![image](https://user-images.githubusercontent.com/31897434/139526776-fad37f98-617e-4fb2-becc-d35ab6312033.png)


**_Hint_**: Try to check when `key = 'Enter'` 

`if (e.key === 'Enter') {...}`

Reply:
This issue is well addressed. Great job! :clap: :clap: